dist/ng-wig.js

```diff
@@ -4,102 +4,102 @@
-        scope.toolbarButtons = ngWigToolbar.getToolbarButtons(attrs.buttons && string2array(attrs.buttons));
-
-        function string2array(keysString){
-          return keysString.split(',').map(Function.prototype.call, String.prototype.trim);
-        }
-
-        scope.toggleEditMode = function () {
-          scope.editMode = !scope.editMode;
-        };
-
-        scope.execCommand = function (command, options) {
-          if(scope.editMode ) return false;
+    .directive('ngWig', ["ngWigToolbar", function (ngWigToolbar) {
 
-          if (command === 'createlink') {
-            options = prompt('Please enter the URL', 'http://');
-            if(!options) {
-              return;
+        return {
+            scope: {
+                content: '=ngWig'
+            },
+            restrict: 'A',
+            replace: true,
+            templateUrl: 'ng-wig/views/ng-wig.html',
+            link: function (scope, element, attrs) {
+
+                scope.editMode = false;
+                scope.autoexpand = !('autoexpand' in attrs) || attrs['autoexpand'] !== 'off';
+                scope.toolbarButtons = ngWigToolbar.getToolbarButtons(attrs.buttons && string2array(attrs.buttons));
+
+                function string2array(keysString){
+                    return keysString.split(',').map(Function.prototype.call, String.prototype.trim);
+                }
+
+                scope.toggleEditMode = function () {
+                    scope.editMode = !scope.editMode;
+                };
+
+                scope.execCommand = function (command, options) {
+                    if(scope.editMode ) return false;
+
+                    if (command === 'createlink') {
+                        options = prompt('Please enter the URL', 'http://');
+                        if(!options) {
+                            return;
+                        }
+                    }
+                    scope.$emit('execCommand', {command: command, options: options});
+                };
             }
-          }
-          scope.$emit('execCommand', {command: command, options: options});
-        };
-      }
-    }
-  }]
+        }
+    }]
 );
 
 
 angular.module('ngWig')
-  .directive('ngWigEditable', function () {
-    function init(scope, $element, attrs, ngModelController) {
-      var document = $element[0].ownerDocument;
+    .directive('ngWigEditable', function () {
+        function init(scope, $element, attrs, ngModelController) {
+            var document = $element[0].ownerDocument;
+
+            $element.attr('contenteditable', true);
+
+            //model --> view
+            ngModelController.$render = function () {
+                $element.html(ngModelController.$viewValue || '');
+            };
+
+            //view --> model
+            function viewToModel() {
+                ngModelController.$setViewValue($element.html());
+                //to support Angular 1.2.x
```

Comment:
this part was removed. 

may be it's easier to create new clean PR? not to resolve a lot of conflicts